wcomponents-core/src/main/java/com/github/bordertech/wcomponents/WTabSet.java

```diff
@@ -496,6 +496,19 @@ public void setActiveTab(final WComponent content) {
 		}
 	}
 
+	/**
+	 * Sets the active tab.
+	 *
+	 * @param tab the active tab.
+	 */
+	public void setActiveTab(final WTab tab) {
+		int index = getTabIndex(tab);
+
+		if (index != -1) {
+			setActiveIndex(index);
+		}
+	}
+
```

Comment:
What do you think...to be consistent and avoid code duplication, all the new `WTab` methods could just call the existing `content` methods like you did with `isActive` method.
```
public void setActiveTab(final WTab tab){
    setActiveTab(tab.getContent());
}
```

Reply:
Sounds good. I'll update to do this.